layouts/shortcodes/version.html

```diff
@@ -0,0 +1,4 @@
+
+<div class="v-container" {{ with .Get "id" }}id="{{.}}"{{ end }}>
+    {{.Inner | markdownify }}
+</div>
```

Comment:
`version`is very generic. Better to call this shortcode `content-version` ?

Reply:
Fixed. Changed to `content-version-container`